Formula/osinfo-db-tools.rb

```diff
@@ -0,0 +1,35 @@
+class OsinfoDbTools < Formula
+  desc "Tools for managing the libosinfo database files"
+  homepage "https://libosinfo.org/"
+  url "https://releases.pagure.org/libosinfo/osinfo-db-tools-1.9.0.tar.xz"
+  sha256 "255f1c878bacec70c3020ff5a9cb0f6bd861ca0009f24608df5ef6f62d5243c0"
+  license "GPL-2.0-or-later"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "json-glib"
+  depends_on "libarchive"
+  depends_on "libsoup"
+  depends_on "python@3.9"
+
+  uses_from_macos "pod2man" => :build
+  uses_from_macos "libxml2"
+
+  def install
+    mkdir "build" do
+      system "meson", *std_meson_args, ".."
+      system "ninja", "install", "-v"
+    end
+  end
+
+  def post_install
+    share.install_symlink HOMEBREW_PREFIX/"share/osinfo"
```

Comment:
When is `HOMEBREW_PREFIX/"share/osinfo"` created?

Reply:
As above, it's created by `osinfo-db` (upon which `libosinfo` depends).